admin/organizations.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Organizations
+description: Learn about managing organizations.
+---
+
+Organizations are groups that tie together users, workspaces, and images. You
+must assign all of your images and workspaces to a specific organization. An
+end-user can only access images that are assigned to the same organization they
+are.
+
+> Be sure to familiarize yourself with the [types of
+> roles](access-control/organizations.md) you can assign users within an
+> organization.
+
+## The default organization
+
+When you first set up Coder, you'll generate the default organization. You can
+then assign users and their workspaces to that organization.
+
+There must always be a default organization, but you can change the one set as
+the default once you have two or more organizations.
```

Comment:
There can be more than one default. So there must always be at least one, and you can set/unset which one(s) are default.